localisation/00_zuwc_united_localisation.csv

```diff
@@ -2988,10 +2988,10 @@ not_importrant_place_tooltip;Is NOT an important place in event chains;;Ist KEIN
 legion_is_destroyed_tooltip;The Burning Legion has NOT fallen;;;;;;;;;;;;;x
 RULE_OPTION_INFERTILE;Infertile;;;;;;;;;;;;;x
 RULE_GENERATE_FAMILIES_INFERTILE_DESC;Most infertile landed historical characters with no recorded family will have a spouse and up to four children generated for them upon starting a new game. This reduces the risk of an early game over.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+convert_to_local_culture;Convert to [Root.Capital.Culture.GetName];;;;;;;;;;;;;x
```

Comment:
@DukeJeans, I would rather not mention culture since it's not clear why you can convert to Nazja, but not to other cultures.
```suggestion
convert_to_local_culture;Convert to Local Culture;;;;;;;;;;;;;x
```

Reply:
This is vanilla loc, still want to change?

Reply:
Vanilla loc is `Convert to Local Culture ([Root.Capital.Culture.GetName])` and, IMHO, the part about `Local Culture` is the most importrant.

Reply:
Aight